content/source/docs/cloud/sentinel/import/tfrun.html.md

```diff
@@ -42,6 +47,58 @@ to only enforce the policy on non-development workspaces is more appropriate.
 [run-glossary]: /docs/glossary.html#run
 [workspace-glossary]: /docs/glossary.html#workspace
 
+## Namespace: root
+
+The **root namespace** contains data associated with the current run.
+
+### Value: `created_at`
+
+* **Value Type:** String.
+
+The `created_at` value within the [root namespace](#root-namespace) specifies the time that the run was created.
+
+### Value: `message`
+
+* **Value Type:** String.
+
+The `message` value within the [root namespace](#root-namespace) specifies the message that is associated with the run.
+
+The default value is *"Queued manually via the Terraform Enterprise API"*.
+
+### Value: `speculative`
+
+* **Value Type:** Boolean.
+
+The `speculative` value within the [root namespace](#root-namespace) specifies whether the plan associated with the run is a [speculative plan](../../run/index.html#speculative-plans) only.
+
+### Value: `is_destroy`
+
+* **Value Type:** Boolean.
+
+The `is_destroy` value within the [root namespace](#root-namespace) specifies whether the run will perform a destroy operation.
```

Comment:
Is this true if _any_ resource is being destroyed? Or only if everything is being destroyed as part of some cleanup/workspace destruction? I originally assumed it was the latter but the copy here makes me think it's if anything in the plan is being destroyed.

Reply:
@glenngillen good catch,. I use the same description that we use in the API docs which indicates that it is the latter example you have called out

Reply:
@glenngillen pushed some changes, please review and resolve conversation if happy with changes. Thanks.